tests/lib/tools.py

```diff
@@ -181,7 +181,6 @@ def setUp(self):
         self.tmp_dir = tempfile.mkdtemp(prefix=prefix)
         self.test_file = 'test_file'
 
-        conf.get_autoupdate_enabled = Mock(return_value=True)
```

Comment:
Removed global mock as tests are extended agenttest setup. This mock getting in the way for other tests especially conf tests which are importing the value from config file